server/channels/api4/plugin.go

```diff
@@ -276,7 +276,9 @@ func getWebappPlugins(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	w.Write(js)
+	if _, err := w.Write(js); err != nil {
+		c.Logger.Warn("Error while writing js response", mlog.Err(err))
```

Comment:
The response is not Java script :wink:

```suggestion
		c.Logger.Warn("Error while writing response", mlog.Err(err))
```

Reply:
Done!